src/renderer/renderElement.ts

```diff
@@ -453,31 +453,33 @@ const generateElementShape = (
           getDiamondPoints(element);
         if (element.strokeSharpness === "round") {
           shape = generator.path(
-            `M ${topX + (rightX - topX) * 0.25} ${
-              topY + (rightY - topY) * 0.25
-            } L ${rightX - (rightX - topX) * 0.25} ${
-              rightY - (rightY - topY) * 0.25
+            `M ${topX + getCornerRadius(rightX - topX, element)} ${
+              topY + getCornerRadius(rightY - topY, element)
+            } L ${rightX - getCornerRadius(rightX - topX, element)} ${
+              rightY - getCornerRadius(rightY - topY, element)
             }
             C ${rightX} ${rightY}, ${rightX} ${rightY}, ${
-              rightX - (rightX - bottomX) * 0.25
-            } ${rightY + (bottomY - rightY) * 0.25}
-            L ${bottomX + (rightX - bottomX) * 0.25} ${
-              bottomY - (bottomY - rightY) * 0.25
+              rightX - getCornerRadius(rightX - bottomX, element)
+            } ${rightY + getCornerRadius(bottomY - rightY, element)}
+            L ${bottomX + getCornerRadius(rightX - bottomX, element)} ${
+              bottomY - getCornerRadius(bottomY - rightY, element)
             }
             C ${bottomX} ${bottomY}, ${bottomX} ${bottomY}, ${
-              bottomX - (bottomX - leftX) * 0.25
-            } ${bottomY - (bottomY - leftY) * 0.25}
-            L ${leftX + (bottomX - leftX) * 0.25} ${
-              leftY + (bottomY - leftY) * 0.25
+              bottomX - getCornerRadius(bottomX - leftX, element)
+            } ${bottomY - getCornerRadius(bottomY - leftY, element)}
+            L ${leftX + getCornerRadius(bottomX - leftX, element)} ${
+              leftY + getCornerRadius(bottomY - leftY, element)
             }
             C ${leftX} ${leftY}, ${leftX} ${leftY}, ${
-              leftX + (topX - leftX) * 0.25
-            } ${leftY - (leftY - topY) * 0.25}
-            L ${topX - (topX - leftX) * 0.25} ${topY + (leftY - topY) * 0.25}
+              leftX + getCornerRadius(topX - leftX, element)
+            } ${leftY - getCornerRadius(leftY - topY, element)}
+            L ${topX - getCornerRadius(topX - leftX, element)} ${
+              topY + getCornerRadius(leftY - topY, element)
+            }
             C ${topX} ${topY}, ${topX} ${topY}, ${
-              topX + (rightX - topX) * 0.25
-            } ${topY + (rightY - topY) * 0.25}`,
-            generateRoughOptions(element, true),
+              topX + getCornerRadius(rightX - topX, element)
+            } ${topY + getCornerRadius(rightY - topY, element)}`,
+            generateRoughOptions(element),
```

Comment:
dropping `preserveVertices` was unintentional, right?

Reply:
Ah yes, sorry about that, fixed!